vars/Debian.yml

```diff
@@ -1,9 +1,16 @@
 ---
+pip_executable: pip3
+python_executable: python3
+ansible_python_interpreter: /usr/bin/python3
```

Comment:
ansible_* variables are special variables and should not be hardcoded in role.

Reply:
fixed